.github/workflows/build-lint-test.yml

```diff
@@ -8,11 +8,17 @@ jobs:
     name: Prepare
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v4
       - name: Use Node.js
         uses: actions/setup-node@v4
         with:
-          node-version-file: '.nvmrc'
+          node-version: 'lts/*'
+      - name: Install Yarn
+        run: corepack enable
+      - uses: actions/checkout@v4
+      - name: Use Node.js and install dependencies
+        uses: actions/setup-node@v4
+        with:
+          node-version: 'lts/*'
```

Comment:
It looks like `corepack` may be already installed on `ubuntu-latest`? I'm reviewing solutions others have posted in the `actions/setup-node` ticket, specifically https://github.com/actions/setup-node/issues/480#issuecomment-1915448139, and it seems like we could do this without having to install Node twice:

``` suggestion
      - uses: actions/checkout@v4
      - name: Install Yarn via Corepack
        run: corepack enable 
      - name: Use Node.js and install dependencies
        uses: actions/setup-node@v4
        with:
          node-version: '.nvmrc'
```

Reply:
I was also traversing the chain of various attempts at adding Corepack support to `setup-node`, and apparently according to [this comment](https://github.com/actions/setup-node/pull/482#pullrequestreview-1017248195) Yarn is already installed on GitHub runners somehow, so we may wish to remove it so that the shim that Corepack installs doesn't conflict:

``` suggestion
      - uses: actions/checkout@v4
      - name: Uninstall existing Yarn
        run: npm -g uninstall yarn
      - name: Install Yarn via Corepack
        run: corepack enable 
      - name: Use Node.js and install dependencies
        uses: actions/setup-node@v4
        with:
          node-version: '.nvmrc'
```

Then again, this doesn't make any sense since it would imply that `npm` is always preinstalled on GitHub runners. Just a suggestion though in case it works.